### PR TITLE
iproute2: Added default config files to separate package

### PR DESCRIPTION
**Pull request by KimBoendergaard**
Yet a recipe I want to be able to customize at build time

### CODE REVIEW

File: recipes/iproute2/iproute2.inc [hunk 1]
```diff
@@ -32,5 +32,10 @@ RDEPENDS_${PN}-rtmon += "libc libdl"
 DEPENDS_${PN}-tc += "libc libm libdl"
 RDEPENDS_${PN}-tc += "libc libm libdl"
 
+PACKAGES =+ " ${PN}-config"
```
Comment (kimrhh) [inline recipes/iproute2/iproute2.inc]: don't you need RDEPENDS on this new package somewhere?

**Comment by kimrhh** [comment]
any update on this?

File: recipes/iproute2/iproute2.inc [hunk 2]
```diff
@@ -32,5 +32,12 @@ RDEPENDS_${PN}-rtmon += "libc libdl"
...
+FILES_${PN}-config = "${sysconfdir}/iproute2/*"
+RDEPENDS_${PN} += "${PN}-config"
+
```
Comment (esben) [inline recipes/iproute2/iproute2.inc]: One blank line should be enough.

File: recipes/iproute2/iproute2.inc [hunk 3]
```diff
@@ -32,5 +32,12 @@ RDEPENDS_${PN}-rtmon += "libc libdl"
...
+
 RECIPE_FLAGS = "iproute2_rt_tables"
 RDEPENDS_${PN}-ip:>USE_iproute2_rt_tables = " iproute2"
+
```
Comment (esben) [inline recipes/iproute2/iproute2.inc]: Remove trailing blank lines.

**Comment by kimrhh** [comment]
replaced by #354

**Pull request closed without merging**